nestkernel/connection_manager.cpp

```diff
@@ -1087,8 +1087,8 @@ nest::ConnectionManager::get_connections( DictionaryDatum params ) const
 
 // Helper method, implemented as operator<<(), that removes ConnectionIDs from
 // input deque and appends them to output deque.
-static inline std::deque< nest::ConnectionID >& operator<<(
-  std::deque< nest::ConnectionID >& out,
+static inline std::deque< nest::ConnectionID >&
+join_deques( std::deque< nest::ConnectionID >& out,
```

Comment:
I agree with Janne on the second part. Regarding the name I don't have a strong opinion, I would be fine with `extend` or even something more specific `extend_connectome` that actually describes would be would like to achieve here.